internal/pkg/cli/pipeline_init.go

```diff
@@ -222,108 +219,19 @@ func (o *initPipelineOpts) RecommendedActions() []string {
 }
 
 func (o *initPipelineOpts) askEnvs() error {
-	if len(o.environments) != 0 {
-		return nil
-	}
-	err := o.getEnvs()
-	if err != nil {
-		return err
-	}
-	if err = o.selectEnvironments(); err != nil {
-		return err
-	}
-	return nil
-}
-
-func (o *initPipelineOpts) getEnvs() error {
-	envs, err := o.store.ListEnvironments(o.appName)
-	if err != nil {
-		return fmt.Errorf("list environments for application %s: %w", o.appName, err)
-	}
-	if len(envs) == 0 {
-		return errNoEnvsInApp
-	}
-	o.envs = envs
-	return nil
-}
-
-func (o *initPipelineOpts) selectEnvironments() error {
-	for {
-		promptMsg := pipelineInitAddEnvPrompt
-		promptHelpMsg := pipelineInitAddEnvHelpPrompt
-		if len(o.environments) > 0 {
-			promptMsg = pipelineInitAddMoreEnvPrompt
-			promptHelpMsg = pipelineInitAddMoreEnvHelpPrompt
-		}
-		addEnv, err := o.prompt.Confirm(promptMsg, promptHelpMsg)
+	if len(o.environments) == 0 {
+		envs, err := o.sel.Environments(pipelineSelectEnvPrompt, pipelineSelectEnvHelpPrompt, o.appName, prompt.WithFinalMessage("Environment pipeline will deploy to, in order:"), "[No additional environments]")
```

Comment:
What do you think of `prompt.WithFinalMesasge("Stages:")`?

Reply:
Hmm. How about "Stage (in order):"
I feel like it should be singular since it'll precede each chosen env. 
Do we need to be explicit that this is the order of the deployment? The help prompt states that they're added sequentially, but I don't think it's made obvious anywhere else.

Reply:
Or "Pipeline deployment stage (in order):"?

Reply:
ok how about this bonkers idea:
```go
o.sel.Environments(pipelineSelectEnvPrompt, pipelineSelectEnvHelpPrompt, o.appName, func(order int) prompt.Option {
  return prompt.WithFinalMessage(fmt.Sprintf("%s stage:", humanize.Ordinal(order))
})
```
See https://github.com/dustin/go-humanize#ordinals

The method signature can be:
```go
Environments(prompt, help, app string, finalMsgFunc func(int) prompt.Option) ([]string, error)
```

Reply:
omg SO GOOD. I reverted back to indexing the for-loop so I'd have `i` for the order # (though I did like your suggestion, @iamhopaul123).

Reply:
Yeah no worries. Let me know when it is ready to review again.